cms/templates/js/optional-content-editor.underscore

```diff
@@ -0,0 +1,5 @@
+<h3 class="modal-section-title">
```

Comment:
This whole editor is a `h3` tag. The heading could simply be `Completion` instead. See the `cms/static/templates/grading-editor.underscore` as an example.

Reply:
modified in the other PR to look like this.
![optional-editor](https://github.com/openedx/edx-platform/assets/19278837/e10cec7b-7958-4338-b385-32f474a70bba)